pkg/profiles/profiles.go

```diff
@@ -0,0 +1,135 @@
+package profiles
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/SUSE/connect-ng/internal/util"
+)
+
+type Result = map[string]interface{}
+
+var profileFilePath = "/run/suseconnect/"
+
+// Profile struct is for handling lage data blobs
+// ID: an unique index to reference Profile data
+//
+//	Profile data can be any type (string, array
+//
+// Data: Can be any data such array of strings, a single large string,a map, etc.
+type Profile struct {
+	ID   string `json:"profileId"`
```

Comment:
Since profile tags follow a snake case format, could we maybe do the same here? So `profile_id` and `profile_data` for the JSON export.